### DISCUSSION TITLE
Run SafeSettings as Github Action

### DISCUSSION THREAD

**Discussion by cheld** [github/safe-settings#505] [Q&A]
Hi all,

I am new to SafeSettings. I tested it locally a bit, but I haven't deployed it yet.

I have a simple use case: running SafeSettings only by a cron job every hour seems sufficient. I'm considering running SafeSettings in a GitHub action every hour, authenticating as a GitHub App.

The SafeSettings would need to perform a full sync and then exit. I wouldn't use the cron configuration of SafeSettings since the cron is controlled by the GitHub Action.

My questions:

Are there any non-obvious reasons not to do this (e.g., exposing secrets in logs)?
Is it possible to run SafeSettings with a full sync and then exit?
Assuming the functionality doesn't already exist, would a pull request be appreciated? (I think the change is only a few lines)

Thanks in advance

**Comment by cep21** [comment]
Hi!

Did you get anywhere with this solution? Curious what progress you made.
